gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -492,6 +493,17 @@ def test_pipeline_read_single_file(self):
       assert_that(pcoll, asserts.count_equals_to(len(_SAMPLE_TEXT_LINES)))
       pipeline.run()
 
+  def test_pipeline_read_all_single_file(self):
+    with TempDir() as tempdir:
+      file_name = self._create_temp_vcf_file(_SAMPLE_HEADER_LINES +
+                                             _SAMPLE_TEXT_LINES, tempdir)
+      pipeline = TestPipeline()
+      pcoll = (pipeline
+               | 'Create' >> beam.Create([file_name])
+               | 'Read' >> ReadAllFromVcf())
+      assert_that(pcoll, asserts.count_equals_to(len(_SAMPLE_TEXT_LINES)))
+      pipeline.run()
```

Comment:
These 6 lines are repeated in each test case. Consider moving them to a function like `_assert_read_all_line_count_equal` that takes a file pattern and line count.

Reply:
Done! Good idea!